app/backend/controllers/events.js

```diff
@@ -0,0 +1,85 @@
+const request = require('request');
+const url = "https://api.tradingeconomics.com/calendar/country/all?c=guest:guest";
+
+/*
+  Get method for events.
+  Using 3rd party API, it saves events to database.
+*/
+module.exports.getEventsFromAPI = async (req, res) => {
+  let Event = req.models['Event']
+
+  request(url, (error, response, body) => {
+    // If there is an error
+    if(error){
+      res.status(400).send({
+        errmsg: error,
+      });
+    }
+
+    // If there is no error, returns event lists 
+    else{
+      const events = JSON.parse(body);
+            
+      events.forEach((ev) => {
+        let event = new Event({
+          ...ev
+        });
+
+        event.save().then(doc => {
+          console.log(doc);
+        }).catch(err => {
+          console.log(err);
+        });
+      });
+    }
+
+    res.status(204).send();
+  })
+}
+
+/*
+  Get method for events.
+  It returns events from database.
+*/
+module.exports.getEvents = async (request, response) => {
+  let Event = request.models['Event']
+  let Country = request.query.country
```

Comment:
Some comment lines would be useful here, as it's not obvious what country and importance variables are supposed to do here and after those lines.